perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/ResourcesManagerBlImpl.java

```diff
@@ -356,8 +355,18 @@ public boolean isUserAllowed(PerunSession sess, User user, Resource resource) {
 	}
 
 	@Override
-	public boolean isGroupAssigned(PerunSession sess, Group group, Resource resource) {
-		return getResourcesManagerImpl().isGroupAssigned(sess, group, resource);
+	public boolean isGroupAssigned(Group group, Resource resource) {
```

Comment:
Please, add PerunSession argument to these methods. We usually include it even if it is not used.